### ISSUE TITLE
 Add webhooks for when registration is known to be created

### ISSUE THREAD

**Issue by mooreds** [FusionAuth/fusionauth-issues#1129]
### Problem

Lots of times we'll want to modify a registration just after it has been created because we want to add roles to be picked up in the JWT.
 
### Solution

Fire a webhook when we absolutely are positive a registration has been created, and that any API calls against this registration will succeed.

### Alternatives/workarounds

Some documented here: https://github.com/FusionAuth/fusionauth-site/pull/462 but none solve the problem.

### Additional context

https://github.com/FusionAuth/fusionauth-issues/issues/1128

### How to vote
Please give us a thumbs up or thumbs down as a reaction to help us prioritize this feature. Feel free to comment if you have a particular need or comment on how this feature should work.

**Comment by glen-84** [comment]
@mooreds Should this issue be closed? `user.registration.create.complete` now exists.

**Comment by mooreds** [comment]
@glen-84 you are correct. I updated our [release notes](https://github.com/FusionAuth/fusionauth-site/pull/1316) and will close this out. 

Thanks!
